curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/welcome-aboard.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 6552939414fdaa21fc734788
+title: Welcome Aboard
+challengeType: 19
+dashedName: welcome-aboard
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+# --description--
+
+The expression `Welcome aboard` is used to greet someone who has just joined a group, team, or organization. It's a warm way of saying "We're glad you're here with us now".
+
+# --question--
+
+## --text--
+
+Listen to the dialogue and select the correct meaning of `Welcome aboard`.
+
+## --answers--
+
+`Maria is happy Tom visited California.`
+
+### --feedback--
+
+Incorrect. The phrase doesn't comment on Tom's visit to California.
+
+---
+
+`Maria is greeting Tom on a boat.`
+
+### --feedback--
+
+Incorrect. The dialogue isn't about being on a boat.
+
+---
+
+`Maria is welcoming Tom to the team.`
+
+---
+
+`Maria is greeting Tom on a boat.`
```

Comment:
This answer doesn't match the feedback. Should it mention directors?

Reply:
Nice catch - yea this answer appears twice it look like - so maybe this is the wrong text for this answer

Reply:
Thanks, Oliver! Just fixed that.